src/wp-includes/theme-templates.php

```diff
@@ -202,6 +203,7 @@ function the_block_template_skip_link() {
 	}() );
 	</script>
 	<?php
+	wp_print_inline_script_tag( str_replace( array( '<script>', '</script>' ), '', ob_get_clean() ) );
```

Comment:
@westonruter Could we consider doing something like this instead. 

```
        $skip_link_script = '
               ( function() { ...
               }() );
        ';
        $handle = 'skip-link-script';
        wp_register_script( $handle, false );
	wp_add_inline_script( $handle, $skip_link_script );
	wp_enqueue_script( $handle );
```

This is similar to what i do here. https://github.com/WordPress/wordpress-develop/pull/4824

Using the normal enqueueing system make sense here. In other places, not so much.

Reply:
Good idea! Done in 205b8deb6dac25994e8994922cb1c091bc68f7f2.

Note that I'm retaining the use of output buffering because there is PHP inline to the script.